packages/block-library/src/pullquote/test/transforms.native.js

```diff
@@ -9,7 +9,7 @@ import {
 	getBlockTransformOptions,
 } from 'test/helpers';
 
-const block = 'Pullquote';
+const block = 'Pullquote \\(deprecated\\)';
```

Comment:
```suggestion
const block = 'Pullquote (deprecated)';
```

I haven't checked , but do we need `\\`?

Reply:
We need `\\` because the tests use the `getBlock` function, which involves string templating with RegExp. Without `\\`, `(` is treated as part of the RegExp, not a character. Using `\\(` ensures it is treated as a normal `(`.

https://github.com/WordPress/gutenberg/blob/e5e95ffdb4765853113b7daf74fc65251b637d6e/test/native/integration-test-helpers/get-block.js#L10-L14

Reply:
I see, thanks for the explanation 👍